core/src/main/scala/io/finch/ToService.scala

```diff
@@ -99,15 +75,41 @@ object ToService {
       private[this] val underlying = endpoints.head.handle(respond400OnErrors)
 
       def apply(req: Request): Future[Response] = underlying(Input.fromRequest(req)) match {
-        case EndpointResult.Matched(rem, out) if rem.route.isEmpty => out.map(oa => conformHttp(
-          oa.toResponse(trA, trE),
-          req.version,
-          includeDateHeader,
-          includeServerHeader
-        )).run
+        case EndpointResult.Matched(rem, out) if rem.route.isEmpty =>
+          val accept = req.accept.flatMap(a => Accept(a))
```

Comment:
I'm wondering if we should keep it under a flag for now? Something like `Bootstrap.configure(negotiateContentType=true)` (disabled by default) so we can avoid header's query for applications where it might affect performance (think of TechEmpower benchmark that might get some performance penalty for `req.accept`).